### PR TITLE
Explicitly list settings that are not synced

### PR DESCRIPTION
**Pull request by szuend**

### CODE REVIEW

**Review by chybie — APPROVED** [review]

File: site/en/docs/devtools/customize/index.md [hunk 1]
```diff
@@ -75,11 +75,12 @@ To update DevTools settings sync:
     {% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/yhIipqtEvDuy6ygB677t.png", alt="Chrome profile sync", width="400", height="579" %}
 2. The DevTools sync settings can be updated via **Settings** > **Sync** > **Enable settings sync** checkbox.
     {% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/LUwFNTDyP22L1euSGg73.png", alt="DevTools sync settings", width="800", height="654" %}
-3. Not all settings are synced. For example, the following **appearance** settings are synced so you have a consistent experience across devices and don’t need to re-define the same settings again.
+3. Most of the settings you see in the **Settings** UI are synced, except for the **Workspace**, **Experiments**, **Devices** tab and a few other general settings. The state of the **Enable settings sync** checkbox is synced across devices as well.
```
Comment (jpmedley) [inline site/en/docs/devtools/customize/index.md:78]:
```suggestion
3. Most of the settings are synced, except for the **Workspace**, **Experiments**, and **Devices** tabs and a few other general settings. The state of the **Enable settings sync** checkbox is synced across devices as well.
```

Comment (jpmedley) [reply site/en/docs/devtools/customize/index.md:78]: >you see in the **Settings** UI 

We know this from the beginning of the sentence and from the context.

File: site/en/docs/devtools/customize/index.md [hunk 2]
```diff
@@ -75,11 +75,12 @@ To update DevTools settings sync:
...
+  For example, the following **appearance** settings are synced so you have a consistent experience across devices and don’t need to re-define the same settings again.
```
Comment (jpmedley) [inline site/en/docs/devtools/customize/index.md:79]:
```suggestion
Additionally, the following **appearance** settings are synced so you have a consistent experience across devices and don’t need to re-define the same settings again.
```

**Review by jpmedley — CHANGES_REQUESTED** [review]
Thank you for the submission. This adds useful information; however I have some suggestions.

File: site/en/docs/devtools/customize/index.md (hunk 2 above)
Comment (chybie) [reply site/en/docs/devtools/customize/index.md:79]: This line is an explanation of the previous line. We mentioned some settings are sync, some aren't. Here are the examples of sync/not sync.

**Review by jpmedley — APPROVED** [review]

**Pull request merged**